### ISSUE TITLE
The logout button should be a button

### ISSUE THREAD

**Issue by phacks** [theodo/falco#143]
Thanks @AlbericTrancart for the report!

From https://a11yproject.com/checklist/#section-content:

> Buttons are used to submit data or perform an on-screen action which does not shift keyboard focus.

It’s currently a `<span>`.

**Comment by MathildeDuboille** [comment]
This has been solved (see issue #146)
